Automattic/jetpack
# About Page: update Gravatar hashes

#### Changes proposed in this Pull Request:

* Let's replace Gravatar hashes for 2 automatticians who are not working for Automattic anymore.

2 folks volunteered to replace them:
- @bisko (`1ec3571e0201a990ceca5e365e780efa`)
- @sergeymitr (`9d6da11cb8f8bd2b0a845816ed9c3390`)

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

* Go to Jetpack > About
* See some new faces.

#### Proposed changelog entry for your changes:

* N/A
